common/log/tag/tags.go

```diff
@@ -336,6 +336,11 @@ func Address(ad string) Tag {
 	return newStringTag("address", ad)
 }
 
+// Env return tag for runtime environment
+func Env(env string) Tag {
```

Comment:
Where do you use this?

Reply:
This will be used in authorizer implementation. In this case, it's uber internal server.